nss-day-cohort-46/holiday-road-forever-genesis
# Dm settup

# Description

Added image for the dependency chart

Fixes # the issue of not having a dependency chart

## Type of change: Documentation



# Testing Instructions

```
git fetch -- all
git checkout dm-settup
````

or check that image is appearing in our repo



# Checklist:
